bluedonkey.py

```diff
@@ -10,40 +16,42 @@ def start_mjpg_streamer():
         "input_opencv.so -r 640x480 --filter /usr/lib/mjpg-streamer/cvfilter_py.so --fargs " + os.path.realpath(__file__),
         "-o",
         "output_http.so -p 8090 -w /usr/share/mjpg-streamer/www"],
-        stdin=subprocess.PIPE,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE)
+        stdin=subprocess.PIPE
+        #, stdout=subprocess.PIPE
+        #, stderr=subprocess.PIPE                   #GIVE ME OUTPUT
+        )
 
 if __name__ == "__main__":
     start_mjpg_streamer()
 
 def init_filter():
     # Display link to stream and dashboard
-    print("Finding IP address...")
-    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-    s.connect(("8.8.8.8", 80))
-    ip_addr = s.getsockname()[0]
-    s.close()
-    if not ip_addr:
-        ip_addr = "localhost"
-    print("Open http://" + str(ip_addr) + ":8090/?action=stream for video stream")
-    print("Open http://" + str(ip_addr) + ":1880/ui for dashboard")
-    print("Run bluedonkey_listen.sh to listen for messages")
+    #ip_addr = "localhost"
+    #print("Open http://" + str(ip_addr) + ":8090/?action=stream for video stream")
+    #print("Open http://" + str(ip_addr) + ":1880/ui for dashboard")
+    #print("Run bluedonkey_listen.sh to listen for messages")
 
     # Redirect input/output to a socket
-    SOCK_OUT = 3001
-    SOCK_IN = 3002
-    sock_out = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-    sock_out.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-    sock_out.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
-    sock_out.connect(("255.255.255.255", SOCK_OUT))
-    sys.stdout = sock_out.makefile('w', buffering=None)
+    #STD_OUT = 3003
+    
+    #SOCK_IN = 3002
+    
+    #std_out = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    
+    #sock_out.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+    
+    #std_out.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
+    #std_out.connect(("", STD_OUT))
+    #sys.stdout = std_out.makefile('w', buffering=None)
+    
```

Comment:
I'm now a little confused what this does. Can you just go ahead and remove the old lines and document what this is supposed to do now?

Reply:
We will go ahead and remove the old lines.